templates/long.yaml

```diff
@@ -5,9 +5,32 @@
 
 - name: Test Project - With Folders
   type: Project
+  acl:
+  - principal:
```

Comment:
If our design goal for the long template is to represent the [Synapse API](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/ResourceAccess.html), then we might want to only support `principal_id`. We can delegate support for team names to the short template, whereby the principal ID would be added in during expansion as long as the team already exists. 

That said, we could still include team information in the long template as top-level items (similar to how we envision new teams will look like). This would be optional for template authors, but they might want to include this for their own reference. I imagine that this would ultimately get ignored by the deployment code since all it needs is the principal ID. 

I've included a preview of what I mean below: 

```
- name: Test Project - With Folders
  type: Project
  acl:
  - principal_id: 3379097
    access_type:
    - READ
    - DOWNLOAD
    - UPDATE
    - CREATE
    - DELETE
    - CHANGE_SETTINGS
    - MODERATE
    - CHANGE_PERMISSIONS
  children:
  - name: Test Folder - With Subfolder and One of Many
    type: Folder
    acl:
    - principal_id: 3330862
      access_type:
      - READ
      - DOWNLOAD
    children:
    - name: Test Subfolder - Empty
      type: Folder
  - name: Test Folder - Empty and One of Many
    type: Folder

- name: Test Project - Empty
  type: Project

- name: synapseformation-test-user
  type: User
  id: 3426116

- name: SynapseFormation Test Team
  type: Team
  id: 3426115
```

Reply:
Agree with this.